### PR TITLE
gaussian prior

### PR DESCRIPTION
**Pull request by XingLLiu**
1. Created new file `test_gaussian_prior.R` for testing gaussian prior on the gaussian genz family. Please do **not** run gaussian prior with `integrationMain.R` as it's not updated. See #62 for implementation details. To run:
`
Rscript test_gaussian_prior.R 1 500 4 rbf 1 uniform
`

2. Fixed merge conflict arisen from commit "consistent gaussian prior".

### CODE REVIEW

File: src/BARTBQ.R [hunk 1]
```diff
@@ -269,7 +269,7 @@ BARTBQSequential <- function(dim, trainX, trainY, numNewTraining, FUN, sequentia
     if (measure == "uniform"){
   	  candidateSet <- replicate(dim, runif(candidateSetNum))
     } else if (measure == "gaussian") {
-      candidateSet <- replicate(dim, rtnorm(candidateSetNum, lower=0, upper=1))
+      candidateSet <- replicate(dim, rtnorm(candidateSetNum, mean=0.5, lower=0, upper=1, sd=dim^2))
```
Comment (harrisonzhu508) [inline src/BARTBQ.R]: nah let's do mean 0

File: src/genz/genz.R [hunk 2]
```diff
@@ -406,3 +408,55 @@ mix <- function(xx){
...
+  }
+  
+  dim <- ncol(xx)
+  u=rep(0.5, dim)
+  a <- rep(100/dim^2, dim) 
+  
+  sum <- (xx - u)^2 %*% a^2
+  y <- exp(-sum)
+  
+  return(y / dtnorm(xx, mean = 0.5, lower = 0, upper = 1, sd = dim^2))
```
Comment (harrisonzhu508) [inline src/genz/genz.R]: i was thinking of just doing 

```
# prepare training dataset
if (measure == "uniform") {
  trainX <- replicate(dim, runif(100))
  trainY <- genz(trainX)
} else if (measure == "gaussian") {
  trainX <- replicate(dim, rtnorm(100, lower=0, upper=1))
  trainY <- genz(trainX) / dtnorm(trainX, lower=0, upper=1)
}
```

so we don't need to add anything

Comment (harrisonzhu508) [reply src/genz/genz.R]: in integrationMain.R

**Comment by harrisonzhu508** [comment]
I suggest we close this issue? I will push my version to master

**Comment by XingLLiu** [comment]
Sorry to make the implementation confusing... It should work now. New codes are in `test_gaussian_prior.R`.
To run:

```
Rscript test_gaussian_prior.R 1 30 4 rbf 1 gaussian 
```

**Pull request merged**